plugins/CorePluginsAdmin/vue/src/PluginsTable/PluginsTable.vue

```diff
@@ -0,0 +1,454 @@
+          >{{ name }}</a>
+          <span v-else>{{ name }}</span>
+
+          <span
+            class="plugin-version"
+            :title="plugin.isCorePlugin
+              ? translate('CorePluginsAdmin_CorePluginTooltip')
+              : undefined"
+          >
+            ({{ plugin.isCorePlugin
+              ? translate('CorePluginsAdmin_OriginCore')
+              : plugin.info.version }})
+          </span>
+
+          <span v-if="pluginNamesHavingSettings.indexOf(name) !== -1">
+            <br /><br />
+            <a
+              :href="`${generalSettingsLink}#${name}`"
+              class="settingsLink"
+            >{{ translate('General_Settings') }}</a>
+          </span>
+        </td>
+        <td class="desc">
+          <div class="plugin-desc-missingrequirements">
+            <span v-if="plugin.missingRequirements">
+              {{ plugin.missingRequirements }}
+              <br />
+            </span>
+          </div>
+          <div class="plugin-desc-text">
+            {{ plugin.info.description.replaceAll('\n', '<br/>') }}
+
+            <span
+              v-if="plugin.info?.homepage && matomoUrls.indexOf(plugin.info?.homepage) === -1"
+              class="plugin-homepage"
+            >
+              <a
+                target="_blank"
+                rel="noreferrer noopener"
+                :href="plugin.info.homepage"
+              >
+                ({{ translate('CorePluginsAdmin_PluginHomepage').replaceAll(' ', '&nbsp;') }})
+              </a>
+            </span>
+
+            <div class="plugin-donation" v-if="plugin.info?.donate?.length">
+              {{ translate('CorePluginsAdmin_LikeThisPlugin') }}
+
+              <a @click.prevent class="plugin-donation-link" :data-overlay-id="`overlay-${name}`">
+                {{ translate('CorePluginsAdmin_ConsiderDonating') }}
+              </a>
+
+              <div
+                :id="`overlay-${name}`"
+                class="donation-overlay ui-confirm"
+                :title="translate('CorePluginsAdmin_LikeThisPlugin')"
+              >
+                <p>{{ translate('CorePluginsAdmin_CommunityContributedPlugin') }}</p>
+
+                <p v-html="$sanitize(translate(
+                  'CorePluginsAdmin_ConsiderDonatingCreatorOf',
+                  `<b>${name}</b>`,
+                ))"></p>
+
+                <div class="donation-links">
+                  <a
+                    v-if="plugin.info?.donate?.paypal"
+                    class="donation-link paypal"
+                    target="_blank"
+                    rel="noreferrer noopener"
+                    :href="getPluginDonateLink(name, plugin.info.donate.paypal)"
+                  >
+                    <img src="plugins/CorePluginsAdmin/images/paypal_donate.png" height="30"/>
+                  </a>
+
+                  <a
+                    v-if="plugin.info?.donate.flattr"
```

Comment:
Shouldn't that also be
```suggestion
                    v-if="plugin.info?.donate?.flattr"
```

Reply:
👍